60-steam-input.rules

```diff
@@ -112,3 +112,12 @@ KERNEL=="hidraw*", KERNELS=="*0955:7214*", MODE="0660", TAG+="uaccess"
 
 # Astro C40
 KERNEL=="hidraw*", ATTRS{idVendor}=="9886", ATTRS{idProduct}=="0025", MODE="0660", TAG+="uaccess"
+
+# Saitek Saitek Pro Flight Yoke
+KERNEL=="hidraw*", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="0bac", MODE="0660", TAG+="uaccess"
+KERNEL=="input", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="0bac", MODE="0660", TAG+="uaccess"
```

Comment:
This should be unnecessary if the kernel is correctly identifying your flight stick as a "joystick" (which is used as a catch-all term for all gaming controllers, including flight sticks, steering wheels, rudders and Xbox-style gamepads).

All recent-ish versions of udev query the capabilities of evdev input devices (`/dev/input/event*`) in `/lib/udev/rules.d/60-input-id.rules`, then assign the `uaccess` tag to anything marked with `ID_INPUT_JOYSTICK` in `/lib/udev/rules.d/70-uaccess.rules`.

Also, if this rule was necessary, it would need to be `input*` and not `input` - the device node is actually called something like `/dev/input/event42`, and `/dev/input` is a directory not a device. So I don't think this rule is actually doing anything for you.